metron-analytics/metron-profiler/pom.xml

```diff
@@ -339,6 +339,7 @@
                                 </excludes>
                             </artifactSet>
                             <transformers>
+                                <transformer implementation="org.atteo.classindex.ClassIndexTransformer"/>
```

Comment:
A compiler annotation.  By virtue of having metron-common as a dependency and therefore  ClassIndex as a transitive dependency, you get the compiler annotation run automatically since is in `META-INF/services` ([here](https://github.com/atteo/classindex/blob/master/classindex/src/main/resources/META-INF/services/javax.annotation.processing.Processor) ).

This is an aspect of `javac`, not maven, so it should cross various build mechanisms.